en-US/mozorg/internet-health.lang

```diff
@@ -52,6 +52,7 @@ Learn more
 Discover open innovation
 
 
+# This string is obsolete.  Do not remove.
```

Comment:
Need to remove this comment, the string is used and not obsolete (see https://github.com/mozilla/bedrock/pull/4699)